developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_26.rst

```diff
@@ -57,9 +69,50 @@ App container parameters with pascal case names ``AppName``, ``UserId`` and ``We
+* ``OCP\Util`` loglevel constants were removed (`PR#34329 <https://github.com/nextcloud/server/pull/34329>`_ by `come-nc <https://github.com/come-nc>`_)
+* 3rdparty dependency ``nikic/php-parser`` was removed (`PR#36393 <https://github.com/nextcloud/server/pull/36393>`_ by `kesselb <https://github.com/kesselb>`_)
+* Deprecated ``OCP\AppFramework\Db\Mapper`` was removed. You can easily migrate to ``OCP\AppFramework\Db\QBMapper``, which is doing the same thing using the query builder instead of string based queries. (`PR#34490 <https://github.com/nextcloud/server/pull/34490>`_ by `rullzer <https://github.com/rullzer>`_)
+* Deprecated classes of ``OCP\Dashboard`` were removed (`PR#35966 <https://github.com/nextcloud/server/pull/35966>`_ by `juliushaertl <https://github.com/juliushaertl>`_)
+
+Added APIs
+^^^^^^^^^^
+
+* New ``OCP\Authentication\Token\IProvider`` for authentication providers: Created a new public interface ``OCP\Authentication\Token\IProvider`` with a method invalidateTokensOfUser to invalidate all tokens of a specific user. ``OC\Authentication\Token\Manager`` implements ``OCP\Authentication\Token\IProvider``. (`PR#36033 <https://github.com/nextcloud/server/pull/36033>`_ by `individual-it <https://github.com/individual-it>`_)
+* ``Auto-Submitted`` header for emails: There now is a new method on the ``OCP\Mail\IMessage`` interface ``IMessage::setAutoSubmitted()``. With this method you can specify that an email was an automatic email or response, to allow mail servers to better detect if an out-of-office reply should be sent, better store/filter the emails and so on. Possible values are documented in the ``OCP\Mail\Headers\AutoSubmitted`` interface. (`PR#36033 <https://github.com/nextcloud/server/pull/36033>`_ by `bennet0496 <https://github.com/bennet0496>`_)
+* ``OCP\BackgroundJob\IJobList::getJobsIterator`` method was added (`PR#36073 <https://github.com/nextcloud/server/pull/36073>`_)
+* New ``OCP\BeforeSabrePubliclyLoadedEvent`` event dispatched on public webdav endpoints (it can be used just like ``OCP\SabrePluginEvent`` to inject additional Sabre plugins in apps for example) (`PR#35789 <https://github.com/nextcloud/server/pull/35789>`_)
+
+Deprecated
+----------
+
+* ``OCP\BackgroundJob\IJobList::getJobs`` method was deprecated (`PR#36073 <https://github.com/nextcloud/server/pull/36073>`_)
+* Controller action annotation ``@UseSession`` is deprecated. Use new ``UseSession`` attribute instead (`PR#36363 <https://github.com/nextcloud/server/pull/36363>`_ by `ChristophWurst <https://github.com/ChristophWurst>`_)
+* **Notifications jQuery event deprecated**: The ``OCA.Notification.Action`` event of the notifications app is deprecated in favor of a ``notifications:action:executed`` event-bus event with (`PR#728 <https://github.com/nextcloud/notifications/pull/728>`_ by `nickvergessen <https://github.com/nickvergessen>`_)
+
+Changes for admins
```

Comment:
wrong manual. these changes should go into the admin_manual

Reply:
Where we should put it there? I see only one page with changes [Maintenance and release schedule](https://docs.nextcloud.com/server/stable/admin_manual/release_schedule.html) and it itself links to develop manual.